extensions/positron-r/src/interpreter-settings.ts

```diff
@@ -21,7 +21,7 @@ export function userRHeadquarters(): string[] {
 		return [];
 	}
 	const userHqDirs = customRootFolders
-		.map((item) => untildify(item))
+		.map((item) => path.normalize(untildify(item)))
```

Comment:
These normalizations did not, alone, fix anything. But it seems like a good idea to me to insulate ourselves from idiosyncrasies of how users/admins represent paths in custom settings.